src/utils/zcl_abapgit_hash.clas.testclasses.abap

```diff
@@ -39,4 +40,18 @@ CLASS ltcl_test IMPLEMENTATION.
 
   ENDMETHOD.
 
+  METHOD sha1_raw.
+
+    DATA: lv_sha1  TYPE zif_abapgit_definitions=>ty_sha1,
+          lv_input TYPE xstring.
+
+    lv_input = 'LOREM_IPSUM'.
```

Comment:
suggest also adding a valid test case, the XSTRING data only allows up to "F", hence lv_input becomes empty